### PR TITLE
Jump Search implemented

### PR DESCRIPTION
**Pull request by mondalshubham1**
Please include a summary of the change. Please also include relevant motivation and context (if applicable).

- [y] I have performed a self-review of my code
- [y] My code follows the style guidelines of this project
- [y] I have added comments to hard-to-understand areas of my code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have made corresponding changes to the comments
- [ ] I have made corresponding changes to the README.md

### CODE REVIEW

File: Algorithms/Search/JumpSearcher.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+
+namespace Algorithms.Searches
+{
+    /// <summary>
+    /// TODO.
+    /// </summary>
+    /// <typeparam name="T">TODO. 2.</typeparam>
```
Comment (siriak) [inline Algorithms/Search/JumpSearcher.cs:9]: Codacy found an issue: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6394504)

**Comment by siriak** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- Algorithms/Search/JumpSearcher.cs  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6394504&bid=20737350)

File: Algorithms/Search/JumpSearcher.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                prev = step; 
+                step += Math.Sqrt(n); 
+                if (prev >= n) 
+                    return -1; 
+            } 
+
+            while (item.CompareTo(sortedData[prev]) > 0) 
+            { 
+                prev++; 
+                if (prev == Math.Min(step, n))
```
Comment (siriak) [inline Algorithms/Search/JumpSearcher.cs:38]: Codacy found an issue: [Do not check floating point equality with exact values, use a range instead.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6394504)

File: Algorithms/Search/JumpSearcher.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+
+namespace Algorithms.Searches
+{
+    /// <summary>
+    /// TODO.
```
Comment (siriak) [inline Algorithms/Search/JumpSearcher.cs:7]: Codacy found an issue: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6394504)

**Comment by siriak** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Algorithms/Search/JumpSearcher.cs  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6394504&bid=20737350)

**Comment by siriak** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Algorithms/Search/JumpSearcher.cs  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6394504&bid=20737350)

**Review by mondalshubham1 — COMMENTED** [review]
Issues mentioned by you taken care of. Please check.

**Comment by siriak** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Algorithms/Search/JumpSearcher.cs  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6394504&bid=20737350)

**Comment by mondalshubham1** [comment]
Not able to understand the codcov/patch. What to do next?

**Comment by mondalshubham1** [comment]
Add label hacktoberfest-accepted.

**Review by siriak — CHANGES_REQUESTED** [review]
Read comments and suggestions here:
https://github.com/TheAlgorithms/C-Sharp/pull/146
https://github.com/TheAlgorithms/C-Sharp/pull/185
https://github.com/TheAlgorithms/C-Sharp/pull/192

Also, you need to cover the implementation with tests.
geeksforgeeks.org is a bad source because they often have a lot of bugs and variables are plainly named incorrectly

**Comment by siriak** [comment]
Only PRs that meet baseline requirements and are likely to be merged without significant changes are marked as hacktoberfest-accepted

**Comment by siriak** [comment]
Closing because of no activity

**Pull request closed without merging**